### ISSUE TITLE
[Cooperative Modes Importer] Refined search function

### ISSUE THREAD

**Issue by HerrKnarz** [darklinkpower/PlayniteExtensionsCollection#184]
### Extension name

Cooperative Modes Importer

### Feature Description

At the moment it seems that the extension only finds a game, when the name in Playnite is exactly the same as on Co-Optimus. Because of that many games can't be found. A common example are games with a "&" instead of "and".

If the search on the web site allows for incomplete names missing words like "and" etc, it could be an option to just leave out special characters like & or : for the search.

And when nothing or several games were found, you could provide a manual search box, where the user can change the search term and after a successful search select a game from a list of found options (if possible with mention of the platform for retro games etc).

I think those features would greatly improve the usability.

**Comment by darklinkpower** [comment]
Extension already ignores characters like `:` and spaces when matching so they are not an issue. Only letters and numbers are used, although there are cases when they'll not match with the current solution.

**Comment by darklinkpower** [comment]
Done in https://github.com/darklinkpower/PlayniteExtensionsCollection/commit/ee952dd65c71d322db4faaae62acbca395660cd9

**Comment by HerrKnarz** [comment]
I just tried around with the search box on the web site and the game names in Playnite. It seems the & character is no issue and the search already substitutes it with an "and". For example "Toe Jam & Earl" and "Toe Jam and Earl" both have the same search results. Unfortunately simply leaving it out won't work. Searching for "Toe Jam Earl" won't find anything.

When I try to find "The Adventures of Batman & Robin" or "The Adventures of Batman and Robin", the web site also finds the game, but unfortunately the extension finds nothing. However I'm pretty sure that yesterday it worked with "and" but not "&". That's why I opened the issue. I don't see any difference between both mentioned games. Both are for the Sega Genesis and only one is found, although both are on the site. That's very strange...
